src/flamegraph.cpp

```diff
@@ -680,6 +799,23 @@ void FlameGraph::saveSvg(const QString& fileName) const
     m_rootItem->setBrush(oldBrush);
 }
 
+void FlameGraph::setColorScheme(ColorScheme _scheme)
+{
+    scheme = _scheme;
+
+    std::function<void(FrameGraphicsItem*)> traverseFunction = [&traverseFunction](FrameGraphicsItem* item) {
+        item->setBrush(brush(item->symbol(), BrushType::Hot));
+        for (const auto& child : item->childItems()) {
```

Comment:
```
for (auto *child : std::as_const(item->childItems())) {
```

Reply:
std::as_const is only available since c++17

Reply:
item->childItems() returns an rValue and qAsConst only works on lValues